network_analysis/configure-subnetwork-trace/src/main/java/com/esri/samples/configure_subnetwork_trace/CodedValueListCell.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.samples.configure_subnetwork_trace;
+
+import javafx.scene.control.ListCell;
+
+import com.esri.arcgisruntime.data.CodedValue;
+import com.esri.arcgisruntime.utilitynetworks.UtilityNetworkAttribute;
+
+/**
+ * Shows the name of the UtilityNetworkAttribute in the attribute selection ComboBox.
```

Comment:
Is this description correct? it looks like the type the ListCall takes is `CodedValue` and not UtilityNetworkAttribute.

Reply:
Good spot, it was wrong! Updated to be `CodedValue`